e2etests/report/main.go

```diff
@@ -69,8 +70,17 @@ func main() {
 
 	if !*skipTests {
 		ctx := log.Stderr(context.Background())
-		ctx, cancel := context.WithTimeout(ctx, 2*time.Minute)
```

Comment:
this we want to keep at 2 mins. we shouldn't include tests that run that long

Reply:
it wouldn't affect what tests we add, CI will always run with 2 minutes, this is just for if we are testing something locally